### PR TITLE
Upgrade dependencies and fix errors

### PR DESCRIPTION
**Pull request by reubano**

### CODE REVIEW

**Comment by bedax** [comment]
any news on this? I think it needs accepting or this repo should be marked as unmaintained.

File: lib/index.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
-const path = require('path')
+const Debug = require('debug')
+const { sep, resolve, basename } = require('path')
 const rollup = require('rollup')
 
+const debug = Debug('metalsmith-rollup')
```
Comment (attila) [inline lib/index.js:5]: Let's not add debug support just like this, or if we think it's needed, let's add debugging to all key areas.

File: lib/index.js [hunk 2]
```diff
@@ -1,16 +1,19 @@
-const path = require('path')
+const Debug = require('debug')
+const { sep, resolve, basename } = require('path')
```
Comment (attila) [inline lib/index.js:2]: Unnecessary change.

File: lib/index.js [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
 const rollup = require('rollup')
 
+const debug = Debug('metalsmith-rollup')
+
 function normalizePath (p) {
-  return p.split(path.sep).filter(function filter (q) {
+  return p.split(sep).filter((q) => {
```
Comment (attila) [inline lib/index.js:8]: Unnecessary coding style change

File: package.json [hunk 4]
```diff
@@ -21,19 +21,21 @@
...
-    "metalsmith": "^2.1.0",
+    "debug": "^4.1.1",
+    "metalsmith": "^2.2.0",
     "mocha": "^5.1.1",
     "nyc": "^11.7.1",
     "semantic-release": "^15.1.7",
     "standard": "^11.0.1",
-    "travis-deploy-once": "^4.4.1"
+    "travis-deploy-once": "^4.4.1",
+    "yarn": "^1.13.0"
```
Comment (attila) [inline package.json:38]: Do we really need yarn here? 🤔

**Review by attila — CHANGES_REQUESTED** [review]
Thanks for your contribution. I do not use metalsmith much anymore but happy to see others care! I think the change is good but it needs more work. I understand that this project needs better tooling but let's not mix coding styles and let's not bring in unnecessary dependencies if possible. Will happily re-review again if needed!

**Comment by bedax** [comment]
These comments seem like something that can be fixed after the PR is accepted, given that this repo is currently broken, and the PR is working.

I think it is important to remember that this plugin is listed on the Metalsmith homepage, and so if you are content to leave it in a broken state, you should add a unmaintained notice.

**Comment by attila** [comment]
Thank you for your contribution and continued care towards this project @bedax 
I will try and find some time to go through the changes and sort out the problems with the PR.

**Comment by attila** [comment]
Feel free to fork this repository, I do not have plans to maintain it any longer.

Thank you for your continued care!

**Pull request closed without merging**